nba_ml.ipynb

```diff
@@ -97,10 +114,278 @@
+     },
+     "metadata": {},
+     "execution_count": 5
+    }
+   ],
+   "source": [
+    "# Some temporary quick cleaning until final cleaned version is developed\n",
+    "# Apparently these are 100% null coloumns\n",
+    "df = df.drop('blank2', axis=1) \n",
+    "df = df.drop('blanl', axis=1)\n",
+    "# row number coloumn\n",
+    "df = df.drop('Unnamed: 0', axis=1)\n",
+    "# Taking only first position listed when multiple are listed\n",
+    "df['Pos'] = df['Pos'].str.split('-').str[0]\n",
+    "\n",
+    "# Quick cleaning method for test purposes - remove all rows with null values\n",
+    "df = df.dropna(axis=0)\n",
```

Comment:
This line was a quick fix (i.e. dropping all rows with null values). Change this after update to cleaning